pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -2641,4 +2642,23 @@ public void testDoNotCreateSystemTopicForHeartbeatNamespace() {
         });
     }
 
+    @Test(timeOut = 30000)
```

Comment:
We'd better setup multi individual pulsar clusters, and turn on topic level replication policy for specific topic to test topic replicate policy. 
For example, we setup cluster-A and cluster-B, and configure `persistent://public/default/topic-test` to replicate from cluster-A to cluster-B,  and then write message-1 to topic `persistent://public/default/topic-test` in cluster-A, we can check whether message-1 can be read from topic `persistent://public/default/topic-test` in cluster-B. Furthermore, we and update, remove topic replication policy, check the messages in two clusters.

Reply:
Yes, I will create a test to cover this later.